### PR TITLE
Remove E2E test utils that are now available upstream

### LINKED ISSUE #5746: Remove e2e test utils now available from @wordpress/e2e-test-utils
## Feature description

The following e2e test utility functions created for this plugin have since been added to [`@wordpress/e2e-test-utils`](https://github.com/WordPress/gutenberg/tree/master/packages/e2e-test-utils/src):

`activatePlugin`
`activateTheme`
`deactivatePlugin`
`deleteTheme`
`installTheme`
`themeInstalled` (renamed `isThemeInstalled` in the core package)

I suggest these functions be removed from the plugin and loaded from the `@wordpress` package.

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation brief

* 

## QA testing instructions

* 

## Demo

* 

## Changelog entry

*

### PR DESCRIPTION
**Pull request by pierlon**
## Summary

Fixes #5746

## Checklist

- [x] My pull request is addressing an open issue (please create one otherwise).
- [x] My code is tested and passes existing [tests](https://github.com/ampproject/amp-wp/wiki/Engineering-Guidelines#tests).
- [x] My code follows the [Engineering Guidelines](https://github.com/ampproject/amp-wp/wiki/Engineering-Guidelines) (updates are often made to the guidelines, check it out periodically).

### CODE REVIEW

File: tests/e2e/config/bootstrap.js [hunk 1]
```diff
@@ -200,7 +200,7 @@ beforeAll( async () => {
 	observeConsoleLogging();
 	await setBrowserViewport( 'large' );
 	await page.setDefaultNavigationTimeout( 10000 );
-	await page.setDefaultTimeout( 3000 );
+	await page.setDefaultTimeout( 10000 );
```
Comment (pierlon) [inline tests/e2e/config/bootstrap.js:203]: I realized the cause for most of the recent flaky E2E tests were related to async actions not completing in time (such as installing a theme). Raising the default timeout to 10s seems like a reasonable amount and has solved the flaky tests I sometimes encounter locally.

File: .github/workflows/build-test-measure.yml [hunk 2]
```diff
@@ -199,8 +199,6 @@ jobs:
   e2e-test-js:
     name: 'E2E test: JS'
     runs-on: ubuntu-latest
-    # Tests are a bit flaky at the moment.
-    continue-on-error: true
```
Comment (pierlon) [inline .github/workflows/build-test-measure.yml:203]: Also now making the E2E job a required check in hopes we no longer see frequent flaky tests in the near future.

**Review by westonruter — APPROVED** [review]
❤️

**Comment by westonruter** [comment]
@pierlon For this to be included in the `2.0` branch, will the `@wordpress/e2e-test-utils` dependency need to be updated as well?

**Comment by pierlon** [comment]
@westonruter Yep it would have to be updated to v4.16.0.

**Comment by westonruter** [comment]
Cool. Would you cherry-pick the commits onto a new branch off of `2.0` and make that package upgrade?

**Pull request merged**